configure

```diff
@@ -787,6 +787,7 @@ enable_backtrace
 enable_shared
```

Comment:
I note you've changed `configure` but I think you actually need to change `configure.ac` and then regenerate `configure` from this -- @ZedThree what's the command to do this?

Reply:
`autoreconf`?
Changing `configure` directly will cause the changes to be lost the next time somebody runs autoreconf and adds the changes.

Should we add a test to make sure `configure` is generated from `configure.ac`?
Unfortunately the generated configure is slightly dependend on the version of autoreconf, making the test non-trivial ...